### ISSUE TITLE
cmake-based build broken

### ISSUE THREAD

**Issue by zorun** [savoirfairelinux/opendht#88]
Building with CMake no longer works:

```
-- Found Msgpack: /usr/lib/libmsgpackc.so (found suitable version "1.4.2", minimum required is "1.2") 
CMake Error at CMakeLists.txt:135 (add_subdirectory):
  The source directory

    /tmp/opendht/doc

  does not contain a CMakeLists.txt file.


-- Configuring incomplete, errors occurred!
See also "/tmp/opendht/build/CMakeFiles/CMakeOutput.log".
```

The breakage resumably comes from 803354552fff338d276e0d1c8db92cee9582eb65:

```
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7e5ae1f..7a5d054 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -132,6 +132,7 @@ endif ()
 if (OPENDHT_TOOLS)
     add_subdirectory(tools)
 endif ()
+add_subdirectory(doc)
```

**Comment by sim590** [comment]
Should be fixed in a533807.

**Comment by zorun** [comment]
Indeed, thanks!
